resources/lang/bg.json

```
@@ -5,6 +5,7 @@
     "join_lobby": "Присъединяване към частна игра",
     "single_player": "Самостоятелна игра",
     "instructions": "Инструкции",
+    "settings": "настройки",
```

I wonder where the word nitpicking comes from 
```suggestion
    "settings": "Настройки",
```